random1.py

```diff
@@ -0,0 +1,6 @@
+import random
+import logging
+
+#Generate 5 random numbers between 10 and 30
+randomlist = random.sample(range(10, 30), 5)
+logging.info(f"Random List : {randomlist}")
```

Comment:
**Suggestion:** Encapsulate the random number generation and logging code inside a function. This will make the code more modular and reusable.
```suggestion
def generate_and_log_random_numbers():
    #Generate 5 random numbers between 10 and 30
    randomlist = random.sample(range(10, 30), 5)
    logging.info(f"Random List : {randomlist}")
generate_and_log_random_numbers()
```